src/jdk.httpserver/share/classes/sun/net/httpserver/HttpConnection.java

```diff
@@ -66,11 +66,13 @@
     volatile State state;
 
     public String toString() {
-        String s = null;
+        final var sb = new StringBuilder(HttpConnection.class.getSimpleName());
```

Comment:
Why not simplify into `new StringBuilder(getClass().getSimpleName());` ?

Reply:
Have a marginal preference for HttpConnection.class.getSimpleName(), since it is a constant, but don't object to getClass().

Reply:
Right - but then why not use a string literal? If you're going to call a method to get the class name then I assume you want to get the actual concrete class name... I don't have much objection either way - especially since I suspect there are no subclasses anyway...

Reply:
Given that there are no subclasses and it's not a public class, I'll stick to the original version.